docs/conf.py

```diff
@@ -98,6 +98,13 @@
 #html_theme_path = ['_themes']
 #html_theme = 'mozilla'
 
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
```

Comment:
in https://pypi.python.org/pypi/sphinx_rtd_theme/0.1.8#using-this-theme-locally-then-building-on-read-the-docs there's a little comment above this line which would make sense to copy here